core/src/main/java/com/exadel/aem/toolkit/bundle/customlists/datasource/ItemComponentDatasource.java

```diff
@@ -19,8 +19,11 @@
 import java.util.Iterator;
 import java.util.List;
 import javax.annotation.Nonnull;
+import javax.jcr.query.Query;
```

Comment:
(cosmetic) Something is still wrong with the ordering: com.. org.. com

Reply:
Done.